### ISSUE TITLE
azure network dns record-set add-record with issue

### ISSUE THREAD

**Issue by pablolibo** [Azure/azure-xplat-cli#2989]
CLI Version: **0.10.0**
OS Type: **Docker Ubuntu**
Installation via:  **docker by github repo**

Mode: **ARM**

Environment: **AzureCloud**

**Description:**

Already domain loaded , I can not create a subdomain with a specific ip

`azure network dns record-set create --resource-group PlatDigCloudServices --dns-zone-name cloud.domain.com --type A --ttl 300 --name platdigmonitor02`

`azure network dns record-set add-record --resource-group PlatDigCloudServices --dns-zone-name cloud.domain.com --record-set-name platdigmonitor02 --type A --ipv4-address 10.0.0.31` 

**Output:**

```
info:    Executing command network dns record-set add-record
+ Looking up the dns zone "cloud.domain.com"                                   
+ Looking up the DNS Record Set "platdigmonitor02" of type "A"                 
+ Updating record set "platdigmonitor02"                                       
error:   parameters.recordSet.location cannot be null.
info:    Error information has been recorded to /root/.azure/azure.err
error:   network dns record-set add-record command failed
```

**Error stack trace: ~/.azure/azure.err**

```
2016-07-05T17:37:21.802Z:
{ [Error: parameters.recordSet.location cannot be null.]
  stack: [Getter/Setter],
  __frame: 
   { name: 'updateRecordSet__20',
     line: 478,
     file: '/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js',
     prev: 
      { name: 'addRecord__18',
        line: 438,
        file: '/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js',
        prev: [Object],
        calls: 3,
        active: false,
        offset: 17,
        col: 23 },
     calls: 0,
     active: false,
     offset: 4,
     col: 58 },
  rawStack: [Getter] }
Error: parameters.recordSet.location cannot be null.
  <<< async stack >>>
    at RecordSetOperations.createOrUpdate (/usr/lib/node_modules/azure-cli/node_modules/azure-arm-dns/lib/dnsManagementClient.js:238:23)
    at updateRecordSet__20 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:482:62)
  at updateRecordSet__20 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:482:59)
  at addRecord__18 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:455:24)
  at __75 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/network.js:1565:15)
  <<< raw stack >>>
    at RecordSetOperations.createOrUpdate (/usr/lib/node_modules/azure-cli/node_modules/azure-arm-dns/lib/dnsManagementClient.js:238:23)
    at __$updateRecordSet__20 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:482:62)
    at __tryCatch (/usr/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:480:250)
    at ___ (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:483:65)
    at __$updateRecordSet__20 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:488:79)
    at __tryCatch (/usr/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:480:150)
    at ___ (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:488:182)
    at __$updateRecordSet__20 (/usr/lib/node_modules/azure-cli/lib/commands/arm/network/dnsZone.js:490:159)

```

**Comment by amarzavery** [comment]
@romangromov @MikhailTryakhov - Can one of you take a look at this?

**Comment by MikhailTryakhov** [comment]
Hi @amarzavery 
please assign such issues for me - Roman is on vacation.

Thanks for this issue - it's already fixed and my [PR#2988](https://github.com/Azure/azure-xplat-cli/pull/2988) is merged and would be published in upcoming release 0.10.2
